src/extractors/custom/timesofindia.indiatimes.com/index.js

```diff
@@ -0,0 +1,48 @@
+export const TimesofindiaIndiatimesComExtractor = {
+  domain: 'timesofindia.indiatimes.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  extend: {
+    reporter: {
+      selectors: ['div.byline'],
+      transforms: {},
```

Comment:
You can exclude the `transforms` here if you're not using it.

Reply:
done! :)